frontend/fe_readme.md

```diff
@@ -0,0 +1,2 @@
+# Frontend app
+Nothing for now...
```

Comment:
can you add some instructions how to run it locally? what is needed to install? (npm, etc) -  I'd add it to our docker-compose based off that :)